typescript/src/live/helpers.ts

```
@@ -116,29 +125,87 @@ export function initMicrophoneRecorder(
 
-  const buffer = readFileSync(resolve(filePath))
-    // remove wav header
-    .subarray(44);
-
   const recorder = {
     interval: null as NodeJS.Timeout | null,
     start() {
       let offset = 0;
       this.interval = setInterval(() => {
-        onAudioChunk(buffer.subarray(offset, (offset += chunkSize)));
+        onAudioChunk(audioData.subarray(offset, (offset += chunkSize)));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)